### PR TITLE
Feature completed

### PR DESCRIPTION
**Pull request by vectorlit**
Feature completed from your start, along with a bunch of other changes. Please merge and then submit pull request back to master so your contribution is completed.

This is a super mess of major changes, which we definitely shouldn't have done as one pull request, but who cares. In the future we can target specific changes since we won't need such big changes.

### CODE REVIEW

**Comment by garkenxian** [comment]
cant approve this request, there are merge conflicts,  Please resolve and I will take a look.

Sorry for the delay, been out of town and just got back,  Been insanely busy.

File: ConventionMobile/ConventionMobile/GlobalVars.cs [hunk 1]
```diff
@@ -4,22 +4,50 @@
...
+
+        public readonly static List<Color> ThemeColorsBG = new List<Color>()
+        {
+            //new Color(244/255, 178/255, 52/255),
+            //new Color(252/255, 205/255, 116/255),
+            //new Color(213/255, 36/255, 36/255)
+            new Color(0.9568, 0.6980, 0.2039),
+            new Color(0.9882, 0.8039, 0.4549),
+            new Color(0.8352, 0.1412, 0.1412)
+        };
```
Comment (garkenxian) [inline ConventionMobile/ConventionMobile/GlobalVars.cs:48]: we should come up with a better way to define these guys.  This is very undescriptive.  if nothing else, comment the color with the "color name"

File: ConventionMobile/ConventionMobile/GlobalVars.cs [hunk 2]
```diff
@@ -535,31 +562,40 @@ public static string GenEventAllEventsCustomizableURL
...
+        //    get
+        //    {
+        //        return String.Format(GenEventAllEventsCustomizableURL, yearlyStartingDate.ToString("yyyy-MM-dd't'HH:mm:ss")) + getOption<string>("GenEventAllEventsCountURL", "/numResults");
+        //    }
+        //    set
+        //    {
+        //        setOption("GenEventAllEventsCountURL", value);
+        //    }
+        //}
+
```
Comment (garkenxian) [inline ConventionMobile/ConventionMobile/GlobalVars.cs]: not a fan of commented code (as you know)  lets kill it unless you feel its needed to keep for some reason.

File: ConventionMobile/ConventionMobile/GlobalVars.cs (hunk 1 above)
Comment (vectorlit) [reply ConventionMobile/ConventionMobile/GlobalVars.cs:48]: Yes, I totally agree. The 3 colors are meant to match the general theme of the year, and will therefore also need to be updated yearly. I don't currently have the capacity to supply color information via the web update service. We can add comments for 2019 as dark orange, light orange, and red.

File: ConventionMobile/ConventionMobile/GlobalVars.cs (hunk 2 above)
Comment (vectorlit) [reply ConventionMobile/ConventionMobile/GlobalVars.cs]: Agreed, I forgot to remove it as part of the update, we can just delete

**Comment by vectorlit** [comment]
Regarding the merge - this is merging into your branch (so that we can then turn around and have you submit the same pull request back into master). This is entirely just to give your commits the credit they deserve in the master repo.

Because of the massive amount of changes in this pull, there's no way for me to resolve the merge without write access to your branch. The web tool doesn't allow you to automatically accept the incoming conflicts - but if you use the github desktop tool (or any git tool in Visual Studio or otherwise) you can accept the incoming conflicts (left -> right).

**Pull request closed without merging**